### PR TITLE
DSP-1356 - Update installation.md

### PR DESCRIPTION
**Pull request by krasimira-bozhanova**

### CODE REVIEW

File: _v2_0_0-docs/installation.md [hunk 1]
```diff
@@ -9,26 +9,227 @@ permalink: v2_0_0-docs/installation/
...
+3. Deploy [recommendations-web.war](http://maven.ontotext.com/content/repositories/publishing-releases/com/ontotext/recommend/recommendations-web/2.0.0/recommendations-web-2.0.0.war) to an application container of your choice. We usually use [Apache Tomcat](http://tomcat.apache.org/tomcat-7.0-doc/appdev/installation.html). Some Java system properties need to be set on the container (in the case of Tomcat, they should go in `bin/setenv.sh` or `bin/setenv.bat`, depending on your platform):
 
-	2.2. `-Dcassandra.host=<cassandra-host>`, same as in the Solr setup.
+	3.1. `-Dcom.ontotext.recommend.backend.url=transport://<host1>:<port1>,<host2>:<port2>/<index-name>` - a URL to the backend. The host and port point to an Elastic cluster
+
+	3.2. `-Dcom.ontotext.recommend.store.url=transport://\<host1\>:\<port1\>,\<host2\>:\<port2\>/\<index-name\>` - a URL to the content store. The host and port should point to an Elastic cluster
+
+	3.3 `-Dcom.ontotext.recommend.requestDocAddress` - the base URL to use for requesting articles that do not exist. It should point to the _/enqueue_ endpoint in a _fetch-annotate-web_ deployment
```
Comment (stefan-enev) [inline _v2_0_0-docs/installation.md]: This is obsolete, can be deleted both from here and the code.

File: _v2_0_0-docs/installation.md [hunk 2]
```diff
@@ -9,26 +9,227 @@ permalink: v2_0_0-docs/installation/
...
 This guide is the bare minimum that needs to be done in order to setup a working system. Scalability and high availability are addressed separately.
 
-1. Install and run Cassandra.
-2. [Install Solr 4.7.0](https://cwiki.apache.org/confluence/display/solr/Installing+Solr).
+1. [Install Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/1.5/_installation.html).
+2. Create a `recommendation-reads` index using the following command containing the mappings and settings:
 
-	2.1. In your `solr.solr.home` directory create a new directory named `recommend` and unpack the contents of [`recommendation-solr-core-1.x.y-bin.zip`](http://maven.ontotext.com/content/repositories/publishing-releases/com/ontotext/recommend/recommendations-solr-core/1.2.0/recommendations-solr-core-1.2.0-bin.zip) there.
+<pre><code>
+curl -f -XPUT http://elastic-host:elastic-port/recommendations-reads -d '{
```
Comment (stefan-enev) [inline _v2_0_0-docs/installation.md]: @cponto recommendations-reads or recommendations-reads-1 here?

Comment (cponto) [inline _v2_0_0-docs/installation.md]: If we use aliases, we should use another name for the index -  `recommendation-reads-1` or whatever convention we choose.

Comment (stefan-enev) [inline _v2_0_0-docs/installation.md]: Is the aliases thing documented anywhere else? (It should be here...)

Comment (krasimira-bozhanova) [inline _v2_0_0-docs/installation.md]: We decided to remove the aliases from the documentation for now since we are not making use of them yet.

**Pull request merged**